contracts/GenArt721CoreV3.sol

```diff
@@ -300,27 +302,31 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     }
 
     /**
-     * @notice Sets the hash for a given token ID `_tokenId`.
+     * @notice Sets the hash seed for a given token ID `_tokenId`.
      * May only be called by the current randomizer contract.
      * May only be called for tokens that have not already been assigned a
      * non-zero hash.
      * @param _tokenId Token ID to set the hash for.
-     * @param _hash Hash to set for the token ID.
+     * @param _hash Hash seed set for the token ID. Only last 12 bytes will be
+     * used.
      * @dev gas-optimized function name because called during mint sequence
      */
     function setTokenHash_8PT(uint256 _tokenId, bytes32 _hash)
```

Comment:
WDYT re: renaming `_hash` to be the more explicit/idiomatic `_hashSeed` instead given this updated paradigm?

Reply:
good call! updated the parameter name to `_hashSeed`. Note that I did consider re-naming the function, but I like keeping this name (and the bytes32 input) as an interface, allowing the core contract to choose how to store and turn the hash seed into a given token's hash.